.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [ ["es2015", {"loose": true}] ],
-  "plugins": ["transform-es3-property-literals", "transform-es3-member-expression-literals", "inline-json"]
+  "presets": [ "es3", ["es2015", {"loose": true}] ],
```

Comment:
It doesn't. It's equivalent to using the es3 plugins but because of when and how plugins and presets are run, having the preset is more reliable for making sure that things are quoted correctly.